### PR TITLE
cEP-0003: Add cib Team

### PR DESCRIPTION
**Pull request by sims1253**
Allow members to propose other members as Team Leaders
Add election and term date
Update the version
Fix typos and grammar

### CODE REVIEW

File: cEP-0003.md [hunk 1]
```diff
@@ -30,15 +30,15 @@ shall have a label corresponding with one of the existing Teams. If a
 member finds an issue without such a label, a fitting one shall be set.
 
 
-Aspects
+Teams
 -------
```
Comment (sils) [inline cEP-0003.md]: correct the heading please

File: cEP-0003.md [hunk 2]
```diff
@@ -80,17 +80,21 @@ for posters, sirts and flyers.
...
+- **cib**:
+The cib team developes and maintains the cib tool for coala.
+
 
 Roles
 -----
 
-A coala team holds the consists of the following roles.
+A coala team consists of the following roles:
+
```
Comment (sils) [inline cEP-0003.md:91]: too many newlines

File: cEP-0003.md [hunk 3]
```diff
@@ -119,10 +124,16 @@ Electing Team Leaders
 Any coala member may propose him or herself to be a Team Leader. The
 proposal should be public and include reasons and plans to take that
 particular team forward. (Archived in the wiki.)
+It is also possible for every coala member to propose another coala
+member as a Team Leader.
```
Comment (sils) [inline cEP-0003.md]: IMO no, teamleaders should make a proper proposal, the initiative must come from them

Comment (sils) [reply cEP-0003.md]: I'd rather include that team leader and member applications are done via the teams repo publically?

Comment (sims1253) [reply cEP-0003.md]: will do

File: cEP-0003.md (hunk 1 above)
Comment (sims1253) [reply cEP-0003.md]: This was done as we now have aspects as our config thing so i wanted to use another word for the teams. And the following list contains teams.

Comment (sils) [reply cEP-0003.md]: my point was that you have to remove a few of the `-` characters

**Comment by sils** [comment]
ack https://github.com/coala/cEPs/pull/41/commits/2b61105c895a9ecc922459024727f4c6fa141f36

**Comment by sils** [comment]
(second ack needed)

**Comment by aptrishu** [comment]
ack 2b61105

**Comment by sils** [comment]
@rultor merge

**Pull request merged**